source/deployment/desktop-app-deployment.rst

```diff
@@ -0,0 +1,112 @@
+
+This page provides a guide on how to 
+
+ - `customize and distribute your own Mattermost Desktop App <XXX>`_
+ - `distribute the official Windows Desktop App silently to end users, pre-configured with the server URL and other app settings <XXX>`_
+
+Custom Build Configuration
+---------------------------
+
+You can customize and distribute your own Mattermost Desktop application by configuring `src/common/config/buildConfig.js <https://github.com/mattermost/desktop/blob/master/src/common/config/buildConfig.js>`_.
+
+1. Configure the Desktop App's `buildConfig.js` file. There are multiple parameters you can configure to customize the user experience:
+
+`defaultTeams`
+~~~~~~~~~~~~~~~~
+
+  Description
+    List of server URLs and their display names added to the desktop app by default, which the user cannot modify. Users can still add servers `through the Server Management page <https://docs.mattermost.com/help/apps/desktop-guide.html#server-management>`_ unless ``enableServerManagement`` is set to false. 
+    
+    Expects an array of key-value pairs.
+
+  Example
+    .. code-block:: none
+
+      defaultTeams: [
+        {
+          name: 'example',
+          url: 'https://example.com'
+        },
+        {
+          name: 'mattermost',
+          url: 'https://www.mattermost.com'
+        }
+      ]
+
+`helpLink`
+~~~~~~~~~~~~~~~~
+
+  Description
+    The URL of the help documentation in Help > Learn More menu bar item. If none specified, the menu option is hidden.
+    
+    Expects a string.
+
+  Examples
+    .. code-block:: none
+
+      helpLink: 'https://about.mattermost.com/default-desktop-app-documentation/'
+      helpLink: ''
+
+`enableServerManagement`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Controls whether users can add, edit or remove servers on the app settings page. If set to false, at least one server must be specified for ``defaultTeams`` or else users cannot interact with any servers.
+    
+    Expects a boolean, true or false.
+
+  Examples
+    .. code-block:: none
+
+      enableServerManagement: true
+
+2. To build the application, follow the `Mattermost Desktop Development Guide <https://github.com/yuya-oc/desktop/blob/master/docs/development.md>`_.
+
+Windows App: Pre-Configuration and Silent Deployment
+------------------------------------------------------
+
+You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL and other app settings.
+
+1. Download the latest Windows installer from the `Mattermost download page <https://about.mattermost.com/download/#mattermostApps>`_.
+
+2. Move the executable file into a shared folder.
+
+3. Create a batch file in Windows
+
+  1. Open a text editor of your choice, such as Notepad or Notepad++.
+  2. Copy and paste the following commands in the text file
+
+    .. code-block:: none
+
+
+      rem "Step 1: Install mattermost desktop into user's local disk"
```

Comment:
Some admins might not be comfortable giving execute permissions to every user for a shared folder. Might be better to copy the exe to the local computer before running it. That way, the shared folder only needs read-only permissions.

Thoughts?

Reply:
Note to self: Installs the Mattermost Desktop App into the user’s local directory. The ``--silent`` parameter is used to not automatically launch the app after the installation is completed.

Reply:
I didn't fix "mattermost" to have a capital letter as it seems this step might be removed?